src/zenml/zen_stores/migrations/versions/8ed03137cacc_polymorthic_run_metadata.py

```diff
@@ -0,0 +1,159 @@
+"""polymorthic run_metadata [8ed03137cacc].
```

Comment:
Depending on the merge order between this PR and [this PR](https://github.com/zenml-io/zenml/pull/2081), we might have to adjust this migration script as well.